lib/tapioca/generator.rb

```diff
@@ -307,6 +308,26 @@ def constantize(constant_names)
       constant_map.values
     end
 
+    sig { params(generator_names: T::Array[String]).returns(T::Array[T.class_of(Compilers::Dsl::Base)]) }
+    def constantize_generators(generator_names)
+      generator_map = generator_names.map do |name|
+        [name, Object.const_get("Tapioca::Compilers::Dsl::#{name}")]
```

Comment:
This is probably a behavior change. 

Previously we were stripping `Tapioca::Compilers::Dsl::` prefix from the supplied generator name and matching it to the names of the generators that we have. That meant that if someone had a `Foo::Bar::BazGenerator` class defined, and they want to just run it, they would be able to do `--generators "Foo::Bar::BazGenerator"`. In this new way, that won't work since we unconditionally assuming the namespace.

We could fix that by looking up by the `Tapioca::Compilers::Dsl::` prefix first and if that fails, falling back to looking up the given name, and if that also work, falling back to treating it as `nil`.

What do you think about that?

Reply:
Ah you're right! I had thought about that use-case (supplying your own compiler) but for some reason thought that `gsub` code wouldn't have supported it, but you're right that it would've.

I was also thinking along the same lines of falling back to trying to find the passed generator in the global namespace. I'll update this code to do that.